psi4/driver/driver_findif.py

```diff
@@ -25,20 +25,129 @@
 #
 # @END LICENSE
 #
+"""Plan, run, and assemble QC tasks to obtain derivatives by finite difference of lesser derivatives.
+
+======
+FINDIF
+======
+
+-----------------------------------
+FiniteDifferenceComputer.__init__()
+-----------------------------------
+* collect findif_stencil_size, findif_step_size from initializer kwargs
+* BaseComputer.__init__()
+* negotiate safety and user wishes on translation and rotation projection
+
+    gradient_from_energies_geometries()
+    -----------------------------------
+    hessian_from_gradients_geometries()
+    -----------------------------------
+    hessian_from_energies_geometries()
+    ----------------------------------
+
+        _geom_generator()
+        -----------------
+
+            _initialize_findif()
+            --------------------
+            * initialize CdSalcs, partition them per irrep, apply user irreps
+
+        * start the governing dict findifrec with parameters, size, mol
+        * for each irrep, for each relevant salc ...
+
+            _displace_cart()
+            ----------------
+            * form new geometry by linear combination
+
+        * ... and collect geometry into field displacements.<label>
+        * for (2, 0) also collect off-diagonal displacements
+        * also collect undisplaced geometry into field reference
+        * return findifrec
+
+* form AtomicComputers for each displacement, particularly changing mol and driver, and possibly relaxing disp symm
+* form dict task_list with keys findifrec labels and vals AtomicComputers
+
+--------------------------------------
+FiniteDifferenceComputer.build_tasks()
+--------------------------------------
+* pass
+
+----------------------------------
+FiniteDifferenceComputer.compute()
+----------------------------------
+* compute() for each job in task list
+
+------------------------------------------
+FiniteDifferenceComputer.get_psi_results()
+------------------------------------------
+
+    Computer.get_results()
+    ----------------------
+
+        Computer._prepare_results()
+        ---------------------------
+        * get_results() for each job in task list
+        * arrange atres data into e/d/g/h fields as available on each of reference and displacements entries
```

Comment:
Unclear what `atres` is.

Reply:
`atres` is a common variable name I choose for `qcel.models.AtomicResult` like `atin` for `AtomicInput`. I've written it out, now.